### PR TITLE
Adds 2 new title songs

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

To add 2 new lobby songs for Yogstation

### Why is this change good for the game?

I think these songs are pretty cool, and will help improve the selection we have. Let me know how you feel about them.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds 2 new lobby songs

### What should players be aware of when it comes to the changes your PR is implementing?

That there will be 2 new lobby songs

### What general grouping does this PR fall under? 

Lobby songs

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No

:cl:  
rscadd: Rainworld THS and Streets of Rogue 5-1 to the lobby songs
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
man after i filled out that entire thing i just realized it looks retarded

**Review by JamieD1 — APPROVED** [review]
